nixos/modules/security/setuid-wrappers.nix

```diff
@@ -107,6 +107,10 @@ in
             chmod 0000 ${wrapperDir}/${program} # to prevent races
             chown ${owner}.${group} ${wrapperDir}/${program}
             chmod "u${if setuid then "+" else "-"}s,g${if setgid then "+" else "-"}s,${permissions}" ${wrapperDir}/${program}
+
+            # compatibility with systemd.services.*.path
+            mkdir -p ${wrapperDir}/bin
+            ln -fs ${wrapperDir}/${program} ${wrapperDir}/bin/${program}
```

Comment:
Wouldn't it be cleaner to just rename the wrapper directory to `/run/setuid-wrappers/bin` or something like that?